typedload/dataloader.py

```diff
@@ -233,7 +239,18 @@ def load(self, value: Any, type_: Type[T]) -> T:
                 self.frefs[tname] = type_
 
         func = self.handlers[index][1]
-        return func(self, value, type_)
+        try:
+            return func(self, value, type_)
+        except TypedloadException as e:
+            e.trace.insert(0, TraceItem(value, type_, annotation))
```

Comment:
Nice, the trace will be in the most intuitive order then :+1:

Reply:
:)